x/nft/testutil/app.yaml

```diff
@@ -1,3 +1,5 @@
+golang_bindings:
```

Comment:
Can remove this from `x/nft` if we don't like it, but I wanted to ensure this change was working end to end, and it seems to be.